### ISSUE TITLE
1.3.1: The EN 301 549 added a note for non-web documents we should consider including in WCAG2ICT

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#702]
EN 301 549 clause 10.1.3.1 says:
>## 10.1.3.1	Info and relationships
>Where ICT is, or includes, a [non-web document](https://github.com/w3c/wcag2ict/issues#NonWebDocument),
the non-web document it shall satisfy the [WCAG 2.2 Success Criterion 1.3.1 Info and Relationships](https://www.w3.org/TR/WCAG22/#info-and-relationships).
>**NOTE:** Where non-web documents contain non-standard structure types (roles), it is best practice to map them to a standard structure type as a fall-back solution for the reader.

WCAG2ICT doesn't have a non-web document specific note. Should WCAG2ICT include this note?

**Comment by loicmn** [comment]
+1 to add the EN 301 549 note, as it has received no comments in the official commenting period of the current draft.

**Comment by mapluke** [comment]
Also +1 from me (after the PR has been created).

**Comment by maryjom** [comment]
The WCAG2ICT TF reached a [resolution](https://www.w3.org/2025/07/03-wcag2ict-minutes.html#50f9) to add the note, as-is to SC 1.3.1. This change has been merged into the editors draft.
